.circleci/config.yml

```diff
@@ -0,0 +1,45 @@
+version: 2.1
```

Comment:
@gfigueroa Circle CI flow is not part of this repo. It should be the project specific repo